### PR TITLE
submit nbsphinx

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
ping @patricksnape @jakirkham @ocefpaf: I think all tests have passed, do you all see anything else that might need to be modified?

File: recipes/nbsphinx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
```
Comment (ocefpaf) [inline recipes/nbsphinx/meta.yaml]: You can probably remove all but `python` and `setuptools` from the `requirements/build` section.

File: recipes/nbsphinx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - python
+    - docutils
+    - jinja2
+    - nbconvert
+    - nbformat
+    - sphinx
+test:
+  imports:
+    - sys
```
Comment (ocefpaf) [inline recipes/nbsphinx/meta.yaml]: This test will always pass :smile: 

Can you add a real test here?

File: recipes/nbsphinx/meta.yaml (hunk 1 above)
Comment (ericmjl) [inline recipes/nbsphinx/meta.yaml]: Ok! As usual, commenting out first to see how it goes before committing to delete. Unlike pymc3, this time round if the tests pass, could you let me remove them from the recipe before merging?

File: recipes/nbsphinx/meta.yaml (hunk 2 above)
Comment (ericmjl) [inline recipes/nbsphinx/meta.yaml]: Haha, yes I knew that one would always pass, but I wasn't quite sure how to test [nbsphinx](https://github.com/spatialaudio/nbsphinx/).

I'll put back `import nbsphinx` first, see whether that one passes or not.

File: recipes/nbsphinx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - setuptools
+
+  run:
+    - python
+    - docutils
+    - jinja2
+    - nbconvert
+    - nbformat
+    - sphinx
+test:
```
Comment (jakirkham) [inline recipes/nbsphinx/meta.yaml]: Please space these out.

File: recipes/nbsphinx/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - nbsphinx
+
+about:
+  home: http://nbsphinx.rtfd.org/
+  license: MIT License
```
Comment (jakirkham) [inline recipes/nbsphinx/meta.yaml]: Please shorten to `MIT`

Comment (jakirkham) [inline recipes/nbsphinx/meta.yaml]: Please add `license_file: LICENSE`.

**Comment by jakirkham** [comment]
Test failure is interesting. I think `nbsphinx` is missing an import.

```
===== testing package: nbsphinx-0.2.9-py27_0 =====
import: 'nbsphinx'
Traceback (most recent call last):
  File "/staged-recipes/build_artefacts/test-tmp_dir/run_test.py", line 26, in <module>
    import nbsphinx
  File "/opt/conda/envs/_test/lib/python2.7/site-packages/nbsphinx.py", line 549, in <module>
    class NotebookError(sphinx.errors.SphinxError):
AttributeError: 'module' object has no attribute 'errors'
TESTS FAILED: nbsphinx-0.2.9-py27_0
```

File: recipes/nbsphinx/meta.yaml (hunk 3 above)
Comment (ericmjl) [inline recipes/nbsphinx/meta.yaml]: NTS: fixed.

File: recipes/nbsphinx/meta.yaml (hunk 4 above)
Comment (ericmjl) [inline recipes/nbsphinx/meta.yaml]: NTS: fixed.

**Comment by ericmjl** [comment]
Yay, all tests passed!

**Pull request merged**